src/main/java/edu/greenblitz/robotName/Robot.java

```diff
@@ -49,13 +50,14 @@ public enum RobotType {
 
 	@Override
 	public void robotInit() {
-		Pathfinding.setPathfinder(new LocalADStar());
+//		Pathfinding.setPathfinder(new LocalADStar());
```

Comment:
return this

Reply:
need this to check before the merge,  will return right after CR from mentor